src/main/webapp/app/course/manage/course-management.service.ts

```diff
@@ -488,6 +488,7 @@ export class CourseExerciseService {
             res.body.forEach((exercise: T) => {
                 exercise.releaseDate = exercise.releaseDate != null ? moment(exercise.releaseDate) : null;
                 exercise.dueDate = exercise.dueDate != null ? moment(exercise.dueDate) : null;
+                exercise.assessmentDueDate = exercise.assessmentDueDate != null ? moment(exercise.assessmentDueDate) : null;
```

Comment:
```suggestion
                exercise.assessmentDueDate = exercise.assessmentDueDate ? moment(exercise.assessmentDueDate) : null;
```
I think you don't need an extra comparison to null here, since if it is null it's evaluated to false

Reply:
Meaning the moment(null) evaluates to false anyways, right? 
I agree on that! So in order to keep it consistent I should then change it for all three dates I guess.